mslice/models/cut/matplotlib_cut_plotter.py

```diff
@@ -10,25 +10,46 @@
 class MatplotlibCutPlotter(CutPlotter):
     def __init__(self, cut_algorithm):
         self._cut_algorithm = cut_algorithm
+        self.axes = None
+        self.canvas = None
 
     def plot_cut(self, selected_workspace, cut_axis, integration_start, integration_end, norm_to_one, intensity_start,
                  intensity_end, plot_over):
         x, y, e = self._cut_algorithm.compute_cut_xye(selected_workspace, cut_axis, integration_start, integration_end,
                                                       norm_to_one)
         integrated_dim = self._cut_algorithm.get_other_axis(selected_workspace, cut_axis)
         legend = self._generate_legend(selected_workspace, integrated_dim, integration_start, integration_end)
-        plt.errorbar(x, y, yerr=e, label=legend, hold=plot_over, marker='o', picker=picker)
-        leg = plt.legend(fontsize='medium')
-        leg.draggable()
         plt.xlabel(self._getDisplayName(cut_axis.units, self._cut_algorithm.getComment(selected_workspace)), picker=picker)
         plt.ylabel(INTENSITY_LABEL, picker=picker)
         plt.autoscale()
         plt.ylim(intensity_start, intensity_end)
+        plt.errorbar(x, y, yerr=e, label=legend, hold=plot_over, marker='o', picker=picker)
+        leg = plt.legend(fontsize='medium')
+        leg.draggable()
         plt.gcf().canvas.manager.add_cut_plot(self)
         if not plot_over:
             plt.gcf().canvas.manager.update_grid()
         plt.draw_all()
 
+    def create_quick_cut(self):
+        self.canvas = plt.gcf().canvas
+        self.axes = self.canvas.figure.gca()
+        plt.gcf().canvas.manager.add_cut_plot(self)
+        self.canvas.manager.set_as_icut()
+        self.canvas.draw()
+
+    def plot_quick_cut(self, workspace, cut_axis, integration_start, integration_end, norm_to_one, intensity_start,
```

Comment:
Is there a reason we don't just reuse the same `plot_cut` method? A little bit of refactoring would make that possible I think and then we wouldn't miss things like the axis labels.